custom_components/flukso/discovery.py

```diff
@@ -381,6 +381,10 @@ def _get_sensor_config(sensor, entry_data, device_info):
             sensorconfig[
                 CONF_VALUE_TEMPLATE
             ] = """{{ (((value.split(",")[1]|round(1)) / 3.3) * 100) | round(2) }}"""
+        elif sensor["type"] == "gas":
+            sensorconfig[
+                CONF_VALUE_TEMPLATE
+            ] = """{{ (value.split(",")[1] / 1000) }}"""
```

Comment:
`value.split(",")[1]` will return a string which cannot be divided

```suggestion
            ] = """{{ ((value.split(",")[1] | float) / 1000) }}"""
```